util/json.go

```diff
@@ -15,7 +16,7 @@ func ReadJSONInto(r io.ReadCloser, data interface{}) error {
 	if err != nil {
 		return errors.Wrap(err, "error reading JSON")
 	}
-	return json.Unmarshal(bytes, data)
+	return errors.Wrapf(json.Unmarshal(bytes, data), "error attempting to unmarshal into %s: %s", reflect.TypeOf(data), string(bytes[:]))
```

Comment:
you can use %T and avoid (explicitly) calling reflect.TypeOf()

Reply:
changed